mautic/mautic
# Adding back social monitoring command

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [y]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [y] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes https://github.com/mautic/mautic/issues/11975



#### Description:

If your try to run the social monitoring command then Mautic response that it does not exist:
```
bin/console mautic:social:monitoring

                                                                   
  There are no commands defined in the "mautic:social" namespace.
...
```
The reason was that those commands were forgotten to be hard-wired when the auto-wiring was introduced and therefore were not auto-configured as commands.

I found 2 other places where this happened and removed the hard-wiring so the auto-configuration could kick in.

All affected commands are tested by functional tests now.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Run `bin/console mautic:social:monitoring` command and it will respond. Please also be sure to check all impacted commands: 

mautic:custom-field:create-column
mautic:import
mautic:reports:scheduler
social:monitor:twitter:hashtags
social:monitor:twitter:mentions
mautic:social:monitoring
